app/etl/ga/concerns/transform_path.rb

```diff
@@ -3,12 +3,25 @@
 module GA::Concerns::TransformPath
   extend ActiveSupport::Concern
 
-  def remove_invalid_prefix
+  def format_events_with_invalid_prefix
     events_with_prefix = Events::GA.where("page_path ~ '^\/https:\/\/www.gov.uk'")
     log(process: :ga, message: "Transforming #{events_with_prefix.count} events with page_path starting https://gov.uk")
     events_with_prefix.find_each do |event|
-      page_path_without_prefix = event.page_path.remove '/https://www.gov.uk'
-      event.update_attributes(page_path: page_path_without_prefix)
+      transformed_attributes = transform_event_attributes(event)
+      event.update_attributes(transformed_attributes)
     end
   end
+
+  def transform_event_attributes(event)
+    sanitised_page_path = event.page_path.remove '/https://www.gov.uk'
+    duplicate_event = Events::GA.find_by(page_path: sanitised_page_path)
+    transformed_attributes = { page_path: sanitised_page_path }
+
+    return transformed_attributes unless duplicate_event
+
+    transformed_attributes[:pageviews] = event.pageviews.to_i + duplicate_event.pageviews.to_i
+    transformed_attributes[:unique_pageviews] = event.unique_pageviews.to_i + duplicate_event.unique_pageviews.to_i
+    duplicate_event.destroy
+    transformed_attributes
+  end
```

Comment:
What about?

1. Use a single exit from the method (remove the `return`)
2. Remove some `to_i` if possible
3. Rename the `transformed_attributes` into `attributes`

Something along the following lines? 
```ruby
  def transform_event_attributes(event)
    sanitised_page_path = event.page_path.remove '/https://www.gov.uk'
    duplicate_event = Events::GA.find_by(page_path: sanitised_page_path)
    attributes = { page_path: sanitised_page_path }

    if duplicate_event
      attributes[:pageviews] = event.pageviews + duplicate_event.pageviews
      attributes[:unique_pageviews] = event.unique_pageviews + duplicate_event.unique_pageviews

      duplicate_event.destroy
    end

    attributes
  end
```